src/pages/email.tsx

```diff
@@ -0,0 +1,80 @@
+import { useState } from 'react';
+import { Button, Container, Input, Text, Textarea } from '@chakra-ui/react';
+
+const Email = () => {
+  const [loading, setLoading] = useState(false);
+  const [message, setMessage] = useState<string>('');
+  const [emailText, setEmailText] = useState('');
+  const [emailRecipients, setEmailRecipients] = useState([]);
+
```

Comment:
I'm not sure what's the best way of integrating TypeScript with useState.
I do believe that being consistent with usage is important. If we want to use `useState<type>`, could we please add them to all the states here?

Reply:
done, removed useState<string>('');